src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  padding: $calendar-paddings;
+  max-width: $calendar-width;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-shrink: 0;
+    width: $day-width;
+    height: $day-height;
+
+    font-family: Arial,
```

Comment:
better add `font-family` to calendar not just for day

` font-family: Arial, serif;`